maxim/logger/components/attachment.py

```
@@ -10,15 +10,15 @@ class FileAttachment:
     name: Optional[str] = None
     size: Optional[int] = None
     mime_type: Optional[str] = None
-    tags: Optional[Dict[str, str]] = None
-    metadata: Optional[Dict[str, Any]] = None
+    tags: Optional[dict[str, str]] = None
+    metadata: Optional[dict[str, Any]] = None
     timestamp: Optional[int] = None
 
     def __post_init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__post_init__`

(ANN204)